next-tavla/src/Admin/scenarios/Edit/index.tsx

```diff
@@ -28,10 +38,23 @@ function Edit({
                 <TavlaButton
                     onClick={() => {
                         setBoardSettings(documentId, settings)
+                        setShowLink(!showLink)
                     }}
                 >
                     Lagre instillinger
                 </TavlaButton>
+                {showLink && (
+                    <div>
+                        <CopyText text={linkURL} />
```

Comment:
Entur sitt designsystem har en [CopyableText-komponent](https://design.entur.no/komponenter/feedback/copyable-text) for dette som vi kan bruke istedet.

Reply:
Vi lagde vårt eget komponent siden CopyableText-komponentet ikke kunne justeres etter riktig bredde

Reply:
Ah skjønner, den er grei 👍